testplan/web_ui/testing/src/AssertionPane/LogGroup.js

```diff
@@ -2,13 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {StyleSheet, css} from "aphrodite";
 
+function getLogColor(logType) {
+    switch(logType) {
```

Comment:
define the logType enumeration in https://github.com/morganstanley/testplan/blob/main/testplan/web_ui/testing/src/Common/defaults.js?